### PR TITLE
Cypress test for newsfeed.

### PR DESCRIPTION
**Pull request by rajputanuj31**
add a test for newsfeed.

### CODE REVIEW

File: tests/System/integration/site/components/com_newsfeed/FeedList.cy.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+describe('Test that the feed view ', () => {
+  it('can display a list of feeds in menu item', () => {
+    cy.db_createNewsFeed({ name: 'automated test feed 1' })
+      .then(() => cy.db_createNewsFeed({ name: 'automated test feed 2' }))
+      .then(() => cy.db_createNewsFeed({ name: 'automated test feed 3' }))
+      .then(() => cy.db_createNewsFeed({ name: 'automated test feed 4' }))
+      .then(() => cy.db_createMenuItem({ title: 'automated test feeds', link: 'index.php?option=com_newsfeeds&view=category&id=5' }))
```
Comment (laoneo) [inline tests/System/integration/site/components/com_newsfeed/FeedList.cy.js]: Why id=5?

File: tests/System/integration/site/components/com_newsfeed/FeedList.cy.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      });
+  });
+
+  it('can display a list of feeds without menu item', () => {
+    cy.db_createNewsFeed({ name: 'automated test feed 1' })
+      .then(() => cy.db_createNewsFeed({ name: 'automated test feed 2' }))
+      .then(() => cy.db_createNewsFeed({ name: 'automated test feed 3' }))
+      .then(() => cy.db_createNewsFeed({ name: 'automated test feed 4' }))
+      .then(() => {
+        cy.visit('index.php?option=com_newsfeeds&view=category&id=5');
```
Comment (laoneo) [inline tests/System/integration/site/components/com_newsfeed/FeedList.cy.js]: Why id=5?

File: tests/System/integration/site/components/com_newsfeed/FeedList.cy.js (hunk 1 above)
Comment (rajputanuj31) [reply tests/System/integration/site/components/com_newsfeed/FeedList.cy.js]: We are testing list of feeds in a particular category so I used `catid = 5` for uncategorized.

Comment (laoneo) [reply tests/System/integration/site/components/com_newsfeed/FeedList.cy.js]: Got it.

**Pull request merged**